packages/frontend/core/src/modules/peek-view/view/image-preview/index.tsx

```diff
@@ -451,7 +451,7 @@ export const ImagePreviewPeekView = (
 ): ReactElement | null => {
   const [blockId, setBlockId] = useState<string | null>(props.blockId);
   const peekView = useService(PeekViewService).peekView;
-  const onClose = peekView.close;
+  const onClose = useCallback(() => peekView.close(), [peekView]);
```

Comment:
Any reasons for this change?

Reply:
`peekView.close()` now will accept a parameter, which will cause `MouseEvent` be passed here.

Reply:
I see. Thanks!